R/error-checks.R

```diff
@@ -138,17 +127,15 @@ isOfLength <- function(object, nbElements) {
 
 #' Check if the provided path has required extension
 #'
-#' @param file file or path name to be checked
-#' @param extension extension of the file required after "."
+#' @param file A single file or path name to be checked.
```

Comment:
"file name or path" maybe? "path name" sounds weird (or I don't get what it means).